metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -57,6 +57,10 @@ case class SbtBuildTool(
     allArgs
   }
 
+  override def cleanupStaleConfig(): Unit = {
+    // no need to cleanup, the plugin deals with that
```

Comment:
Is that an issue with Mill and Maven too?

Reply:
for mill it's not needed (fixed) and for maven it is